src/components/modal/modal.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react'
+import { Button, Modal } from 'react-bootstrap'
+
+function ModalWindow() {
+  const [show, setShow] = useState(false)
+
+  const handleClose = () => setShow(false)
+  const handleShow = () => setShow(true)
+
+  return (
+    <div>
```

Comment:
This component should only contain the Modal component. Open button, state management should be at that component where you want to use the modal. It should receive `show` and `handleClose` as props
You can make the modal component reusable by passing title, bodyText etc. as props.